es-MX/Pergamino_de_Personajes/Informacion_Legal.md

```diff
@@ -1,4 +1,42 @@
-# Información Legal
+# Legal
 
-> **Warning**
-> Esta sección aún no ha sido traducida.
+Este libro es una traducción del Free5e Character's Codex (original: © 2025 by Wyrmworks Publishing, disponible en [https://free5e.com](https://free5e.com)), que tiene licencia Creative Commons 4.0 Atribución/Reconocimiento disponible en [https://creativecommons.org/licenses/by/4.0/legalcode](https://creativecommons.org/licenses/by/4.0/legalcode). Para ver la versión del libro oficial, tanto como la información legal oficial, leer la versión original en inglés disponible en [https://free5e.com](https://free5e.com). Este traducción es el trabajo de un voluntario, Sequoia Sanchini, con al ayudo de la comunidad Free5e y también está disponible con la misma licencia. 
+
+Esta obra contiene materiales extraídas del A5E System Reference Document (A5ESRD/ Documento de Referencia de Sistema A5E) por EN Publishing y disponible en [A5ESRD.com](https://A5ESRD.com), basado en Level Up: Advanced 5th Edition (Nivela: 5a Edición Avanzado), disponible en [www.levelup5e.com](https://www.levelup5e.com).
+A5ESRD tiene la licencia Creative Commons 4.0 Atribución/Reconocimiento disponible en [https://creativecommons.org/licenses/by/4.0/legalcode](https://creativecommons.org/licenses/by/4.0/legalcode).
+Esta obra incluye materiales extraídos del Documento de referencia del sistema 5.1 (“SRD 5.1”) de Wizards of
+the Coast LLC, que está disponible en https://dnd.wizards.com/es/resources/systems-reference-document. 
+El SRD 5.1 tiene licencia Atribución/Reconocimiento 4.0 Internacional disponible en [https://creativecommons.org/licenses/by/4.0/legalcode](https://creativecommons.org/licenses/by/4.0/legalcode).
+Esta obra incluye materiales extraídas del Lazy GM's Resource Document (Documento de Recursos por el GM Perezoso) por Michael E. 
+Shea de [SlyFlourish.com](https://SlyFlourish.com), disponible con la licencia Creative Commons Atribución/Reconocimiento 4.0 Internacional. 
```

Comment:
These should be one line, and the space at the end of what is currently line 11 should be removed to make the linter (format checker) happy.

Reply:
The dot after the "E." in line 10 is not the end of a sentence, it's a marker of an initial. So lines 10 and 11 should be combined.